Automattic/wc-calypso-bridge
# Align wide all block-based checkout pages

### Changes proposed in this Pull Request:



Closes #1042 .



This PR aims to set the page container width to "wide" for the _Cart_ and _Checkout_ pages during a new site's provision process.

##### Screenshots:

| Cart [Before] | Cart [After] |
|---|---|
|![Cart page BEFORE](https://user-images.githubusercontent.com/1105590/224362201-12a50b79-33b3-4264-a952-0fcc1c2f2f59.png)|![screencapture-ecomchlidev-wpcomstaging-cart-2023-03-10-17_40_58](https://user-images.githubusercontent.com/1105590/224362396-3f2c744f-d5eb-46a9-8d1c-139cee271984.png)|

|Checkout [Before]|Checkout [After]|
|---|---|
|![screencapture-ecomchlidev-wpcomstaging-checkout-2023-03-10-17_39_55](https://user-images.githubusercontent.com/1105590/224362495-cb246daf-37af-4c51-9368-4c4ba93b283c.png)|![screencapture-ecomchlidev-wpcomstaging-checkout-2023-03-10-17_40_37](https://user-images.githubusercontent.com/1105590/224362531-8a13f435-b169-4b05-9be4-adeac32e7971.png)|



### How to test the changes in this Pull Request:
- Visit an ecommerce site.
- Remove the `wc_calypso_bridge_one_time_operation_woocommerce_create_pages` option.
- Refresh the admin home.
- Visit the Cart and Checkout pages and ensure that it looks as expected.
